draft-ietf-rats-ar4si.md

```diff
@@ -197,13 +197,13 @@ The following sections detail requirements for these three categories.
 Identity Evidence must be conveyed during the establishment of any trust-based relationship.
 Specific use cases will define the minimum types of identities required by a particular Relying Party as it evaluates Attestation Results, and perhaps additional associated Evidence.
 At a bare minimum, a Relying Party MUST start with the ability to verify the identity of a Verifier it chooses to trust.
-Attester identities may then be acquired through signed communications with the Verifier identity and/or the pre-provisioning Attester public keys in the Attester.
+Attester identities may then be acquired through signed or encrypted communications with the Verifier identity and/or the pre-provisioning Attester public keys in the Attester.
 
-During the Remote Attestation process, the Verifier's identity will be established with a Relying Party via a Verifier signature across recent Attestation Results.
+During the Remote Attestation process, the Verifier's identity must be established with a Relying Party, often via a Verifier signature across recent Attestation Results.
```

Comment:
Should we start capitalize MUSTs? There are several low cap musts in the text and this one caught my attention. It is mandatory that we check all occurrences if is not a "MUST" maybe we MUST use non-normative language that is not "must".

Reply:
I guess it depends what we consider normative from this document.   It certainly is a MUST from the perspective of the architecture document.   We likely should have a multi-person discussion on what to capitalize in this doc.

Reply:
Every must is a MUST and will be fixed to all caps by rfced (if that syntax reaches that gateway function). The multi-person discussion has to happen and we do not have to do that now in this PR.